test/lib/converter.js

```diff
@@ -499,4 +499,55 @@ describe.skip('from swagger to raml: apis-guru', function () {
 			}
 		}
 	});
+});
+
+describe('from raml (URL) to swagger: apis-guru', function () {
+	//TODO: Test with includes that contains includes fail, need to check parser for fix it
+	//TODO: RAML example when converts and activate flag validate get errors
+	let baseURI = 'https://raw.githubusercontent.com/mulesoft/oas-raml-converter/master/test/data/apis-guru/raml/';
+	let baseDir = __dirname + '/../data/apis-guru/raml';
+	let testFiles = fs.readdirSync(baseDir);
+	let testUrls = [];
```

Comment:
Yes, test the examples in RAML10 from the same github repository.
I check in travis, but i get "errors" only in files with big size that i suppose is for memory or timeout.

You can check here:

https://travis-ci.org/unicornio8/oas-raml-converter